src/Company/CompanyUpdateCommand.php

```diff
@@ -455,4 +464,52 @@ public function setNafCode(?string $nafCode): self
 
         return $this;
     }
+
+    /**
+     * @return string|null
+     */
+    public function getMetaTitle(): ?string
+    {
+        return $this->metaTitle;
+    }
+
+    /**
+     * @param string|null $metaTitle
+     */
+    public function setMetaTitle(?string $metaTitle): void
```

Comment:
```suggestion
    public function setMetaTitle(?string $metaTitle): self
```

Pour les setter on return self pour les chaîner. Et par convention, on met les setter en premier

Reply:
Tu veux dire les setters avant les getters? Est-ce que ça a du sens de le faire sur ce fichier où tous les getters sont en premier?
Et peux-tu m'expliquer pourquoi le return $this? Ce que ça change concrètement

Reply:
retourner `$this` permet de faire des appels chaînés : `$object->method1()->method2()->method3()`. si tu ne retournes pas `$this`, tu vas devoir faire : `$object->method1(); $object->method2(); $object->method3();`. dans l'idée, si une méthode n'a rien à retourner, elle doit retourner `$this` pour permettre ces appels chaînés. on n'autorise `void` que pour les tests unitaires, et les surcharges de méthodes qui ont déjà un return void.